### PR TITLE
fix: `types-RC.yaml`

### PR DESCRIPTION
**Pull request by Antonov548**

### CODE REVIEW

File: src/rinterface.c [hunk 1]
```diff
@@ -7449,11 +7449,7 @@ SEXP R_igraph_from_prufer(SEXP prufer) {
 
   SEXP r_result;
                                         /* Convert input */
-  if (!Rf_isNull(prufer)) {
-    R_SEXP_to_vector_int_copy(prufer, &c_prufer);
-  } else {
-    IGRAPH_R_CHECK(igraph_vector_int_init(&c_prufer, 0));
-  }
+  R_SEXP_to_vector_int_copy(prufer, &c_prufer);
```
Comment (krlmlr) [inline src/rinterface.c:7452]: Is this change intended?

Comment (Antonov548) [reply src/rinterface.c:7452]: Yes, this also needed

Comment (Antonov548) [reply src/rinterface.c:7452]: Input argument in this function couldn't be `NULL`

**Comment by krlmlr** [comment]
Reapplied.

We would gain more confidence merging this if we had the SAN builds running in CI/CD.
